library/Fission/User/Mutation.hs

```diff
@@ -47,29 +49,25 @@ create ::
   -> Maybe HerokuAddOnId
   -> UTCTime
   -> Transaction m (Either Error.Create UserId)
-create username password email herokuAddOnId now =
-  hashPassword' password >>= \case
-    Left err ->
-      return (Left err)
-
-    Right secretDigest -> do
-      let newUserRecord = User
-            { userUsername      = username
-            , userEmail         = email
-            , userRole          = Regular
-            , userActive        = True
-            , userHerokuAddOnId = herokuAddOnId
-            , userSecretDigest  = secretDigest
-            , userInsertedAt    = now
-            , userModifiedAt    = now
-            }
+create username did email herokuAddOnId now = do
```

Comment:
hmmm what do we do for heroku accounts? keep basic auth for now? Generate semi-permanent jwts instead? Just thinking out loud here

Reply:
Yeah, let's keep basic auth for them as an option for now. We'll switch them to keys in the future, but we need to figure out exactly how we want to do that. There's a few options, such as asking them for a key in the CLI command, but that's less convenient than generating one for them which is less secure. TL;DR leave it as a password for now :P